bin/pycbc_make_skymap

```diff
@@ -326,17 +352,23 @@ if __name__ == '__main__':
                              'production analyses!')
     parser.add_argument('--gracedb-server', metavar='URL', default=None,
                         help='URL of GraceDB server API for uploading events. ')
-
+    parser.add_argument('--custom-frame-file', type=str, default=None, required=False, nargs='+',
```

Comment:
In the interesting of minimizing the amount of code, there is no need to specify `default=None` and `required=False` here.

Reply:
I have removed them.